### ISSUE TITLE
Translations\Zh_Tw does not comply with psr-4 autoloading standard

### ISSUE THREAD

**Issue by yuvilio** [jimmiw/php-time-ago#88]
FYI, for more recent version 3.1.0, I get this deprecation notice when adding `"jimmiw/php-time-ago": "^3.1.0"`:
` to `composer.json`: 

```
Deprecation Notice: Class Westsworld\TimeAgo\Translations\Zh_Tw located in ./wp-content/plugins/composer-libs/jimmiw/php-time-ago/src/Westsworld/TimeAgo/Translations/Zh_TW.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///home/yuvilio/apps/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

```bash
$ composer --version
Composer version 1.10.5 2020-04-10 11:44:22
$ php --version
PHP 7.4.4 (cli) (built: Apr 27 2020 00:25:38) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
$ 
```

**Comment by jimmiw** [comment]
Thanks for the report, i'll look into it

**Comment by dhutapratama** [comment]
![image](https://user-images.githubusercontent.com/9693486/96606125-1f9a8180-1321-11eb-8817-413b37f8cde1.png)
![image](https://user-images.githubusercontent.com/9693486/96606244-4658b800-1321-11eb-8edb-8f73d0576bf4.png)

**Comment by jimmiw** [comment]
Fixed in 3.1.1

**Comment by yuvilio** [comment]
Thank you!
